components/test_raftstore/src/pd.rs

```diff
@@ -817,8 +817,10 @@ impl TestPdClient {
                 .wait()
                 .unwrap()
                 .unwrap();
-            if now.get_start_key() != region.get_start_key()
-                || now.get_end_key() != region.get_end_key()
+            if (now.get_start_key() != region.get_start_key()
+                && self.get_region(region.get_start_key()).is_ok())
+                || (now.get_end_key() != region.get_end_key()
+                    && self.get_region(now.get_end_key()).is_ok())
```

Comment:
need to care empty end key?

Reply:
oh, thanks for reminding

Reply:
seems unnecessary. If the end_key is changed after a split, the new end_key must be non-empty.